python/ola/Makefile.mk

```
@@ -58,16 +58,42 @@ python/ola/Version.py: python/ola/Makefile.mk configure.ac config/ola_version.m4
 # TESTS
 ##################################################
 
+python/ola/ClientWrapperTest.sh: python/ola/Makefile.mk
+	mkdir -p $(top_builddir)/python/ola
+	echo "export PYTHONPATH=${top_builddir}/python:${top_srcdir}/python; $(PYTHON) ${srcdir}/python/ola/ClientWrapperTest.py; exit \$$?" > $(top_builddir)/python/ola/ClientWrapperTest.sh
```

I don't think you need the export if you drop the `;`
```suggestion
	echo "PYTHONPATH=${top_builddir}/python:${top_srcdir}/python $(PYTHON) ${srcdir}/python/ola/ClientWrapperTest.py; exit \$$?" > $(top_builddir)/python/ola/ClientWrapperTest.sh
```

See also PidStoreTest.sh and https://github.com/OpenLightingProject/ola/blob/master/data/rdm/Makefile.mk#L27

This also avoids corrupting the shell this is run in by overwriting things with export.

Is the top_srcdir bit still needed now you've added all those symlinks via configure?

---

Amusingly the whole export and top_srcdir thing was copied from an existing test, but I wasn't even consistent in using it in all of the cases.

I don't think there's actually any harm in it, as it's being used in shell scripts that I believe are exec'ed from the build process, so it wouldn't inherit elsewhere.  But it's still poor form (particularly on a line with a command) so better to fix that and the top_srcdir.

(and annoyingly I pushed before I realized there was another use of the pattern in tools/rdm...)

---

Heh, it's okay, we seem to often be rather inconsistent.

Thanks for fixing.